bundles/org.openhab.binding.automower/src/main/resources/OH-INF/update/update.xml

```diff
@@ -4,6 +4,334 @@
 	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
 
 	<thing-type uid="automower:automower">
+		<instruction-set targetVersion="2">
```

Comment:
Not sure if this is a requirement, but i appreciate it if the instructions are ordered chronological

Reply:
fixed